### PR TITLE
BNetzA fix

### PR DESCRIPTION
**Pull request by the-infinity**
Fixes the BNetzA import. Also (temporary) fixes an issue with shapely not compiling at GCC, which was introduced at the latest Alpine image. There is a final fix merged to main, but it's not released yet: https://github.com/shapely/shapely/issues/2081

### CODE REVIEW

File: Dockerfile.dev [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.11-alpine
+FROM python:3.11.9-alpine3.20
```
Comment (derhuerst) [inline Dockerfile.dev:1]: Let's use a floating tag to benefit from Python security patches.

```suggestion
FROM python:3.11-alpine3.20
```

Alternatively, if we do fix the Python version, let's set up a bot that updates the tag.

File: Dockerfile.prod [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Builder stage for compiling dependencies
-FROM python:3.11-alpine AS builder
+FROM python:3.11.9-alpine3.20 AS builder
```
Comment (derhuerst) [inline Dockerfile.prod:2]: Same as above.

**Review by derhuerst — CHANGES_REQUESTED** [review]

**Review by flauschzelle — COMMENTED** [review]
I can't judge if the changes to the BNetzA import are correct, because there's no ticket to compare them to. Have you tested them manually?

It would be helpful to have an automated test for the import. If the full dataset is too big, maybe test it with a smaller sample taken from of the actual data?

**Comment by the-infinity** [comment]
@flauschzelle got a biiiiit out of the scope of this MR, but now we have a test, and some other approvements :)

File: Dockerfile.prod (hunk 2 above)
Comment (the-infinity) [reply Dockerfile.prod]: Done

File: Dockerfile.dev (hunk 1 above)
Comment (the-infinity) [reply Dockerfile.dev]: Done

**Review by flauschzelle — APPROVED** [review]

**Pull request merged**